.github/workflows/black.yml

```diff
@@ -0,0 +1,9 @@
+name: Black Formatter
+
+on: pull_request
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: psf/black@stable
```

Comment:
Do we need these for the pre commit?

Reply:
Well it's not needed for the pre-commit but could be used as a safety net in case some developers/contributors forget or don't want to use pre-commit and/or black. This will make the gh actions fail and block the merge of unformatted code!